bin/migration-scripts/0.13.5.js

```diff
@@ -0,0 +1,34 @@
+export const description = 'v0.13.5';
+
+export default (uniPath, content) => {
+  if (!content?.blocks) {
+    return content;
+  }
+
+  content.blocks = adjustBlocks(content.blocks);
```

Comment:
Сделай пожалуйста конструкцию как и в других миграциях
```suggestion
  for (const block of content.blocks) {
    adjustСarousel(block);
  }
```